lib/code_corps/emails/forgot_password_email.ex

```diff
@@ -0,0 +1,20 @@
+defmodule CodeCorps.Emails.ForgotPasswordEmail do
+  import Bamboo.Email
+  import Bamboo.PostmarkHelper
+
+  alias CodeCorps.Emails.BaseEmail
+
+  def create(user, token) do
+    BaseEmail.create
+    |> to(user.email)
+    |> template(template_id(), [link: link(token)])
+  end
+
+  defp template_id, do: Application.get_env(:code_corps, :postmark_forgot_password_template)
+
+  defp link(token) do
+    Application.get_env(:code_corps, :site_url)
+    |> URI.merge("password/reset?token=#{token}")
```

Comment:
I added a comment to your ember PR. You can easily do it without dynamic URL segments. 

That being said, I'm not sure which is better. I feel the query param approach, which is this one here, is more commonly seen with reset tokens. However, I can't argue that the dynamic URL approach, i.e. the one you're proposing, doesn't look cleaner.